tools/monitoring/README.md

```diff
@@ -28,14 +28,21 @@ monitoring_service file to make any updates.
 
 ### Update Image
 
-1. `images/monitoring/Makefile` Commands to build and deploy the monitoring
-   images.
+1. Build and deploy the monitoring images.
+   ```bash
+   cd images/monitoring/
+   make build
+   make push_versioned
```

Comment:
`make push` is sufficient
https://github.com/knative/test-infra/blob/master/shared/Makefile.simple-image